lxd/apparmor/instance_forkproxy.go

```diff
@@ -52,6 +52,11 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   ptrace (read),
   ptrace (trace),
 
+  /etc/machine-id r,
+  {{ .rootPath }}/run/systemd/resolve/stub-resolv.conf r,
+  {{ .rootPath }}/run/{resolvconf,NetworkManager,systemd/resolve,connman,netconfig}/resolv.conf r,
+  /usr/lib/systemd/resolv.conf r,
```

Comment:
That requires the rootPath prefix, no?

Reply:
I looked through the code and saw that in some places with /usr rootPath is used and in some it isn't. I thought that with rootPath it should work fine, so I added it.